.circleci/config.yml

```diff
@@ -99,13 +99,16 @@ jobs:
           name: Install required software
           command: |
             export GOPATH=$HOME/go
-            export PATH=$PATH:$HOME/.cargo/bin:$HOME/janusbin
-            brew install go
-            brew install bats
-            brew install gpg2
+            export PATH=$PATH:$HOME/.cargo/bin:$HOME/janusbin]
             curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain stable
             if [ ! -x ~/janusbin/janus ]; then
               cd $HOME; curl -sL https://raw.githubusercontent.com/ethereumproject/janus/master/get.sh | bash
+              brew update
+              export BDIR="$(brew --repo homebrew/core)" && echo "$BDIR" && git -C "$BDIR" reset --hard master && git -C "$BDIR" fetch origin ac2308d7640e45de0de1dbcbb6d4f42fcdd8adde && git -C "$BDIR" checkout FETCH_HEAD && HOMEBREW_NO_AUTO_UPDATE=1 brew install go && brew switch go 1.9.3 && git -C "$BDIR" checkout master && unset BDIR
```

Comment:
It's a kind of magic.

For now (without updating the cache version) this won't be used anyways. And later, when we decide to update cache version/toolset, we probably update also this code - so my question - is this necessary?

Reply:
Man, I know. _Apparently_ this is the "accepted" way to `brew install` specific versions.